specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2018-07-01/storagesync.json

```diff
@@ -2508,15 +2682,211 @@
+            "description": "Required. Gets or sets the location of the resource. This will be one of the supported and registered Azure Geo Regions (e.g. West US, East US, Southeast Asia, etc.). The geo region of a resource cannot be changed once it is created, but if an identical geo region is specified on update, the request will succeed."
+        },
         "tags": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          },
-          "description": "The user-specified tags associated with the server endpoint."
+            "type": "object",
+            "additionalProperties": {
+                "type": "string"
+            },
+            "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used for viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key with a length no greater than 128 characters and a value with a length no greater than 256 characters."
         },
+        "properties": {}
+    },
+    "required": [
+        "location"
+    ],
+    "description": "The parameters used when creating a storage sync service."
+},
+    "SyncGroupCreateParameters": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+	  ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SyncGroupCreateParametersProperties",
+          "description": "The parameters used to create the sync group"
+        }
+      },
+      "description": "The parameters used when creating a sync group."
+    },
+    "SyncGroupCreateParametersProperties": {
+      "type": "object",
+      "description": "Sync Group Create Properties object.",
+      "properties": {
+       }
+    },
+    "CloudEndpointCreateParameters": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+	  ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CloudEndpointCreateParametersProperties",
+          "description": "The parameters used to create the storage sync service."
+        }
+      },
+      "description": "The parameters used when creating a storage sync service."
+    },
+    "CloudEndpointCreateParametersProperties": {
+      "description": "CloudEndpoint Properties object.",
+      "properties": {
+        "storageAccountResourceId": {
+            "type": "string",
+            "description": "Storage Account Resource Id"
+        },
+        "storageAccountShareName": {
+            "type": "string",
+            "description": "Storage Account Share name"
+        },
+        "storageAccountTenantId": {
```

Comment:
please remind me why is tenantId needed? can the storageAccount be in a different tenant?

Reply:
yup. You're correct. Cloudendpoint creation can point to different StorageAccount Tenant  Id